engine.py

```diff
@@ -451,6 +454,106 @@ def _panels_supported(self):
         # setInterface work properly in houdini 14. If that goes through, we'll
         # be able to make embedded panels work in houdini 14 too.
 
+    def _run_app_instance_commands(self):
```

Comment:
Most of this logic copied from Maya. See delayed execution at bottom of this method for the wacky part.